### PR TITLE
New Parsers Docs

### PR DESCRIPTION
**Pull request by anagrius**

### CODE REVIEW

File: content/integrations/data-shippers/beats/filebeat.md [hunk 1]
```diff
@@ -138,9 +138,10 @@ For example, when sending a web server access log file to Humio, you can use the
 
 ### Parsing JSON data
 
-Humio supports [JSON parsers]({{< ref "json-parsers.md" >}}).
+We DO NOT recommend that you use the JSON parsing built into Filebeat, instead Humio has it's own JSON support.
```
Comment (mwl) [inline content/integrations/data-shippers/beats/filebeat.md]: I'd prefer a notice box instead of capital letters

File: content/parsers/_index.md [hunk 2]
```diff
@@ -5,35 +5,77 @@ date: 2018-03-15T08:19:58+01:00
...
-## Custom Parsers
+## Choosing a parser
 
-If the built-in parsers do not support your data type, then you can create
-your own.
+When sending data to Humio you have to specify:
 
-Humio supports two types of custom parsers:
+1. Which repository to storage the data in
+1. Which parser to use for ingesting the data
```
Comment (mwl) [inline content/parsers/_index.md]: Does this GHFMD of using `1.` repeatedly actually work in Hugo?

File: content/parsers/creating-a-parser.md [hunk 3]
```diff
@@ -1,144 +1,187 @@
...
-When creating a regular expression parser, you can add key-value parsing.
-When you enable key-value parsing, Humio runs an extra parser on the incoming event.
-This extra parser looks for key-value fields of the form:
+We want the parser to produce two events (one per line) and use the timestamp of each line as
+the time at which the event occurred - i.e. assign it to the special `@timestamp` and `@timezone` fields.
 
- * `key=value`
- * `key="value"`
- * `key='value'`
+To do this we could write a parser script like:
```
Comment (mwl) [inline content/parsers/creating-a-parser.md]: There is a parser shortcode which we should extend to also be able to present the new parsers

**Comment by mortengrouleff** [comment]
appears to have been merged...

**Pull request closed without merging**